src/sockperf.cpp

```diff
@@ -2413,14 +2415,22 @@ static void set_select_timeout(int time_out_msec) {
 
 //------------------------------------------------------------------------------
 void set_defaults() {
-#if !defined(WIN32) && !defined(__FreeBSD__)
+#if !defined(__windows__) && !defined(__FreeBSD__)
     bool success = vma_xlio_try_set_func_pointers();
     if (!success) {
         log_dbg("Failed to set function pointers for system functions.");
         log_dbg("Check vma-xlio-redirect.cpp for functions which your OS implementation is missing. "
                 "Re-compile sockperf without them.");
     }
 #endif
+    int rc = 0;
+    if (os_sock_startup() == false) { // Only relevant for Windows
```

Comment:
Should we replace `#endif` above with `#else` and add `#endif` below this condition?

Reply:
Yes agree since `os_sock_startup()` is only relevant for windows